Examples/FastMarchingSegmentation/FastMarchingSegmentation.cs

```diff
@@ -0,0 +1,113 @@
+/*=========================================================================
+*
+*  Copyright Insight Software Consortium
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*         http://www.apache.org/licenses/LICENSE-2.0.txt
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*
+*=========================================================================*/
+
+// This example is based on ITK's FastMarchingImageFilter.cxx example
+
+//INPUTS:  {BrainProtonDensitySlice.png}
+//    OUTPUTS: {FastMarchingImageFilterOutput5.png}
+//    ARGUMENTS:    81 114 1.0  -0.5  3.0   100 100
+
+
+using System;
+using System.Globalization;
+
+using itk.simple;
+using PixelId = itk.simple.PixelIDValueEnum;
+using SitkImage = itk.simple.Image;
+
+namespace itk.simple.examples
+{
+    public class Program
```

Comment:
Possibly change the name of the class from "Program" to something meaningful?

Reply:
done